cfg/config.go

```diff
@@ -58,3 +60,7 @@ func NewDefaultConfig() *Configuration {
 		PodFilter:               "backupPod=true",
 	}
 }
+
+func GetGlobalRepository() string {
+	return fmt.Sprintf("s3:%s/%s", Config.GlobalS3Endpoint, Config.GlobalS3Bucket)
+}
```

Comment:
it doesn't feel right to put this in this package. But I'm not sure where to better put it. Before rebasing it's been in `constants.go`. 
Any idea where it'd fit better?

Reply:
I feel like this function is redundant anyways, see my other comment about the `Backend.String()` function. If there's still need for this function though, the `Backend` type would sound like a good home for this I guess.